.env.development

```diff
@@ -1,2 +1,2 @@
 REACT_APP_API_URL=http://localhost:4000
-REACT_APP_USER_TOKEN=token
+REACT_APP_USER_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsImlhdCI6MTY2MzE1MDMwMiwiZXhwIjoxNjYzMjM2NzAyfQ.CfreVeo8McdzqNteK9FG127P2U1OKlsaqUnNNyekUUc
```

Comment:
I think the `REACT_APP_USER_TOKEN` environment variable is supposed to represent the *key* used when storing the token in local storage, e.g.
`localStorage.setItem(process.env.REACT_APP_USER_TOKEN, theActualToken)`

Best change this back to `token`